test/SampleStream.test.js

```diff
@@ -2,22 +2,13 @@ var reader = require('..');
 var tape = require('tape');
 var fs = require('fs');
 var split = require('split');
-var AWS = require('aws-sdk-mock');
 var path = require('path');
 
-AWS.mock('S3', 'getObject', function (params, callback) {
-    console.log('parameter', params);
-    callback(null, fs.readFileSync(path.join(__dirname, 'fixtures/getObjectResponse.txt')).toString());
-});
-
 function testFunc(r, f, expected, t) {
     var sample = reader.SampleStream({rate: (r * 0.1), filter: f});
 
-    var getStream = require('s3scan').Get('./fixtures/AAAAAAAAAAAAAA.2015-10-19-17.e5b6526a.gz');
-
-    var transformedStream = getStream
-    .pipe(split())
-    .pipe(sample);
+    var splitStream = split();
+    var sampledStream = splitStream.pipe(sample);
```

Comment:
Per chat, decided to remove the `s3scan` part of the streams since we're just testing the sampler stream. Idea is to write N lines of logs to the stream directly and let it sample some portion of the data we put in.